### ISSUE TITLE
Additional Controls for Play and Pause on Slideshow View

### ISSUE THREAD

**Issue by mtribone** [samvera/hyrax#2989]
### Descriptive summary

When I search the catalog and view the Slideshow of the results in Hyrax Version: 2.1.0.rc1, there aren't any controls to pause and play the slideshow and all controls should be keyboard accessible. 

** Note: This functionality comes from [Blacklight Gallery](https://github.com/projectblacklight/blacklight-gallery)

### Rationale

The user is not provided with a means to pause or stop this content, which automatically updates or changes. This can make the content, and sometimes even the entire page, unusable for some users–especially keyboard only users.

[WCAG 2.2.2](https://www.w3.org/TR/2013/NOTE-UNDERSTANDING-WCAG20-20130905/time-limits-pause.html)

> Auto-updating: For any auto-updating information that (1) starts automatically and (2) is presented in parallel with other content, there is a mechanism for the user to pause, stop, or hide it or to control the frequency of the update unless the auto-updating is part of an activity where it is essential.

[WCAG 2.1.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-keyboard-operable.html)

> The intent of this Success Criterion is to ensure that, wherever possible, content can be operated through a keyboard or keyboard interface (so an alternate keyboard can be used). When content can be operated through a keyboard or alternate keyboard, it is operable by people with no vision (who cannot use devices such as mice that require eye-hand coordination) as well as by people who must use alternate keyboards or input devices that act as keyboard emulators.

### Expected behavior

Visible and actionable controls to pause and play the slideshow that are operable via the keyboard. 

### Actual behavior

When starting the slideshow by selecting the slideshow icon and the image to start the slideshow, the slideshow appears over the search results and cycles automatically through the images listed in the results. The slideshow does have arrows to cycle back and forth, but nothing to pause and play. 

### Steps to reproduce the behavior

1. Start from the [Nurax Home Page](https://nurax.curationexperts.com/?locale=en) and perform an empty search that searches all of Nurax
![screen shot 2018-04-30 at 2 59 25 pm](https://user-images.githubusercontent.com/4163828/39444885-2bebf4ea-4c87-11e8-88f2-8c45fae415e4.png)

1. Select the slideshow option
![screen shot 2018-04-30 at 3 00 29 pm](https://user-images.githubusercontent.com/4163828/39444918-4b312938-4c87-11e8-8504-18f6a77ce9cc.png)

1. Select one of the images to start the slideshow
![screen shot 2018-04-30 at 3 02 05 pm](https://user-images.githubusercontent.com/4163828/39444987-807bc418-4c87-11e8-9903-145b48f605f6.png)

1. View the slideshow and try to pause and play the slideshow
![screen shot 2018-04-30 at 3 03 37 pm](https://user-images.githubusercontent.com/4163828/39445071-cee107c6-4c87-11e8-99f0-9e8f0347c4bd.png)

**Comment by rjkati** [comment]
This issue is still present in hyrax 3.0.0.pre.rc3

**Comment by jeremyf** [comment]
In my investigation, we've pinned Blacklight Gallery to `~> 0.7`; The current version is `~> 2.0`, which is incompatible with our version of Blacklight.

I'm going to check about updating the version to the `~> 1.0` series.

_(Issuing a a general lament that our surface area of obligation is too large for the amount of resources that we have to support it.)_

**Comment by jeremyf** [comment]
Welp, until we get Blacklight compatability up to `v7`, we'll be using `~> 0.7`.

Now to look at the particulars.

**Comment by jeremyf** [comment]
@jlhardes I want to double check, is the slideshow feature something we want to continue supporting?

I ask because I'm wondering about the utility and the surface area of feature support given our scrappy coalition of the willing.

**Comment by jlhardes** [comment]
I don't know the use case for adding in the slideshow view of results but what we have implemented right now is buggy for keyboard use and mouse use - the forward arrow is not functioning at all and there is no pause or play buttons. If upgrading Blacklight Gallery improves functionality for keyboard users and mouse users, that would be worth it but it also seems like it could stay in the Backlog until we upgrade Blacklight to v7. This specific problem is also mentioned in #3271 as a reason to upgrade to Blacklight 7.

**Comment by jeremyf** [comment]
@jlhardes  Given #3271, I think this should sink back to the backlog.

**Comment by jlhardes** [comment]
Is it possible to disable the slideshow part of the blacklight_gallery gem we have now, as was discussed in #4763? That would also help this accessibility concern and is possibly what we want long-term for Hyrax anyway - no slideshow view of search results at all. I'm not sure if it's easy to keep the slideshow option turned off after upgrading Blacklight and this gem but it seems like that is the best default setup for Hyrax. If it is better to take the gem as it is for default Hyrax and provide documentation for turning off gem features, then I could see that as being a way to address this now as well. That might be a better path for maintenance purposes.

**Comment by jlhardes** [comment]
Until we can update to Blacklight 7 and then update to the latest version of the blacklight_gallery gem (2.0), the best option we have to offer for this right now is to share how to disable the slideshow view: https://github.com/projectblacklight/blacklight-gallery/wiki/Turn-off-the-slideshow-view. This issue will return to the Backlog and we will evaluate the accessibility concern again after these updates are applied.

**Comment by rjkati** [comment]
This has been fixed in Hyrax 4.0.0 with the upgrade to Blacklight 7
